@stellar/typescript-wallet-sdk/src/walletSdk/Exceptions/index.ts

```diff
@@ -159,6 +159,13 @@ export class WithdrawalTxNotPendingUserTransferStartError extends Error {
   }
 }
 
+export class WithdrawalTxMissingDestinationError extends Error {
+  constructor() {
+    super(`Withdrawal transaction missing withdraw_anchor_account field`);
```

Comment:
Maybe worth noting in this kind of errors that it's an unexpected response from the Anchor side

Reply:
@Ifropc do you mean updating the error copy to something like this?:

Reply:
`Withdrawal transaction missing 'withdraw_anchor_account' field. Anchor should set this field for transactions in 'pending_user_transfer_start' status.`

Reply:
Yeah just a message saying that it's anchor issue. So developer knows where the error comes from. The exact wording is up to you, and maybe doing it in a separate PR is a good idea!

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/164/commits/50d045e7fe368a1514c652126720ed036cd53ff1